pkgs/by-name/cy/cy/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub, nix-update-script, go, }:
```

Comment:
format this file with nixfmt-rfc-style. you need to run the nixfmt from within the devshell